core/src/main/java/com/amplitude/core/utilities/InMemoryResponseHandler.kt

```diff
@@ -12,27 +12,27 @@ internal class InMemoryResponseHandler(
     val eventPipeline: EventPipeline,
     val configuration: Configuration,
     val scope: CoroutineScope,
-    val dispatcher: CoroutineDispatcher,
-    val events: List<BaseEvent>
+    val dispatcher: CoroutineDispatcher
 ) : ResponseHandler {
 
     companion object {
         const val BACK_OFF: Long = 30000
     }
 
-    override fun handleSuccessResponse(successResponse: SuccessResponse) {
-        triggerEventsCallback(events, HttpStatus.SUCCESS.code, "Event sent success.")
+    override fun handleSuccessResponse(successResponse: SuccessResponse, events: Any, eventsString: String) {
```

Comment:
@qingzhuozhen can you explain why the change to move events/eventString to ResponseHandlers from Storage?